src/javascript/ast-value.ts

```diff
@@ -28,6 +28,9 @@ function literalToValue(literal: estree.Literal): LiteralValue {
  */
 function unaryToValue(unary: estree.UnaryExpression): LiteralValue {
   const operand = expressionToValue(unary.argument);
+  if (operand === null || operand === undefined) {
+    return operand;
```

Comment:
Actually we want to match javascript semantics here, even when it looks like a mistake. `+null === 0` so when we encounter that code the value we should produce is `0`.

Recommend changing those three expressions on operator to be like:

`+(operator as any)`